notebooks/book2/03/linreg_post_pred_plot.ipynb

```diff
@@ -0,0 +1,305 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.            fig, ax = plt.subplots()
</pre><p>why line#4 and line#6 are same?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>